### PR TITLE
AC-419: Fill up Troubleshooting

### PR DESCRIPTION
**Pull request by adtserapio**
## Description of what I changed

I have added a guide (troubleshooting.md) file that documents login issues and ways to file issues in OpenMRS Talk and JIRA.

## Issue I worked on

See https://issues.openmrs.org/browse/AC-419
See https://codein.withgoogle.com/dashboard/task-instances/6683284093272064/

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Troubleshooting Android Client
+
+#### Can't Login?
+
+There are a few things you would need to first take note in logging in. Make sure that you enter a valid server for the URL address. Moreover, the username and password does not correspond to your OpenMRS ID. Follow the information on the link [here](https://openmrs.github.io/openmrs-android-client-user-guide/introduction.html) to log into the test server.
```
Comment (f4ww4z) [inline troubleshooting.md]: Can you mention https://wiki.openmrs.org/display/ISM/OpenMRS+environments somewhere here?

File: troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Troubleshooting Android Client
+
+#### Can't Login?
+
+There are a few things you would need to first take note in logging in. Make sure that you enter a valid server for the URL address. Moreover, the username and password does not correspond to your OpenMRS ID. Follow the information on the link [here](https://openmrs.github.io/openmrs-android-client-user-guide/introduction.html) to log into the test server. 
+
+#### Have Another Issue?
```
Comment (f4ww4z) [inline troubleshooting.md]:
```suggestion
#### Have an Issue?
```

File: troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+This guide covers the basic development setup and things to take note off while using the android client. Make sure the guide first. The guide walks through the basics of entering patient details, uploading patient photos, etc, which you could check out [here](https://openmrs.github.io/openmrs-android-client-user-guide/features.html).
```
Comment (f4ww4z) [inline troubleshooting.md]:
```suggestion
This guide covers the basic setup and instructions to take note of while using the Android client. Make sure to refer to the guide first. The guide walks through the basics of entering patient details, uploading patient photos, etc. which you may check out [in this page](https://openmrs.github.io/openmrs-android-client-user-guide/features.html).
```

Comment (f4ww4z) [reply troubleshooting.md]: Note that this is a guide for the users / implementers, not strictly for new developers (although you can say that some instructions are for developers).

File: troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+However, if there are other issues you come across, you could take the following steps.
```
Comment (f4ww4z) [inline troubleshooting.md]:
```suggestion
However, if there are other issues you come across, you could take the following steps:
```

**Review by f4ww4z — CHANGES_REQUESTED** [review]
Thanks @Pilipino ! Please add the new page to SUMMARY.md and address my comments.

**Comment by adtserapio** [comment]
Sorry I did something wrong with the squashing. I'll open a new PR. :)

**Pull request closed without merging**